src/units/user/dbus-broker.service.in

```diff
@@ -13,6 +13,23 @@ Sockets=dbus.socket
 ExecStart=@bindir@/dbus-broker-launch --scope user
 ExecReload=@bindir@/busctl --user call org.freedesktop.DBus /org/freedesktop/DBus org.freedesktop.DBus ReloadConfig
 Slice=session.slice
+LockPersonality=yes
```

Comment:
Make sure to test this with the latest systemd version, as most of these in user units imply PrivateUsers=yes to work